Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml.cs

```diff
@@ -22,18 +22,18 @@ public CairoSettingsWindow()
         {
             InitializeComponent();
 
-            loadThemes();
-            loadLanguages();
-            loadRadioGroups();
-            loadCategories();
-            loadHotKeys();
-            loadLoggingLevels();
+            LoadThemes();
```

Comment:
As above, private methods in Cairo use a lowercase first letter. Please revert the capitalizations